types/constants.go

```diff
@@ -61,7 +61,7 @@ func init() {
 		BlockFrequency = 6                       // 6 seconds: slow enough for developers to see ~each block, fast enough that blocks don't waste time.
 		MaturityDelay = 10                       // 60 seconds before a delayed output matures.
 		GenesisTimestamp = Timestamp(1424139000) // Change as necessary.
-		RootTarget = Target{0, 0, 4}             // Standard developer CPUs will be able to mine blocks.
+		RootTarget = Target{4, 0, 4}             // Standard developer CPUs will be able to mine blocks.
```

Comment:
This would suggest that the developer is getting a few dozen hashes per second.

This is really slow, you are probably hashing an entire block somewhere, or calling ID() a bunch of times or something.